### DISCUSSION TITLE
Welcome to Advice-generator-app Discussions!

### DISCUSSION THREAD

**Discussion by WitchDevelops** [commIT-Group/advice-generator-app#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by car2t** [comment]
Great idea Domi!!!

Better annoy people here. Ha ha ha.
I will create one already. 
Bye for now
